cognite/client/utils/_auxiliary.py

```diff
@@ -177,6 +177,11 @@ def split_into_n_parts(seq: Sequence[T], *, n: int) -> Iterator[Sequence[T]]:
     yield from (seq[i::n] for i in range(n))
 
 
+@overload
+def split_into_chunks(collection: set, chunk_size: int) -> list[list]:
+    ...
+
+
 @overload
 def split_into_chunks(collection: list, chunk_size: int) -> list[list]:
```

Comment:
Can't this be added to the overload below? 

```python
def split_into_chunks(collection: list | set, chunk_size: int) -> list[list]:
```

Reply:
Aye, yeah, will change